fastai/transforms.py

```diff
@@ -533,20 +577,27 @@ class CropType(IntEnum):
     RANDOM = 1
     CENTER = 2
     NO = 3
+    GOOGLENET = 4
 
-crop_fn_lu = {CropType.RANDOM: RandomCrop, CropType.CENTER: CenterCrop, CropType.NO: NoCrop}
+crop_fn_lu = {CropType.RANDOM: RandomCrop, CropType.CENTER: CenterCrop, CropType.NO: NoCrop, CropType.GOOGLENET: GoogleNetResize}
 
 class Transforms():
-    def __init__(self, sz, tfms, normalizer, denorm, crop_type=CropType.CENTER, tfm_y=TfmType.NO, sz_y=None):
+    def __init__(self, sz, tfms, normalizer, denorm, crop_type=CropType.CENTER,
+                 gnet_resize_params={'min_area_frac':0.08, 'min_aspect_ratio':0.75, 'max_aspect_ratio':1.333, 'flip_p':0.5, 'random_state':None},
```

Comment:
Remove gnet_resize_params from everywhere you've included them - if the user wants different params, they can just create the transform list manually. These shortcut functions are really just designed for convenience when using defaults.